pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDParameters.java

```diff
@@ -215,8 +218,11 @@ public PMDConfiguration toConfiguration() {
         configuration.setIgnoreIncrementalAnalysis(this.isIgnoreIncrementalAnalysis());
 
         LanguageVersion languageVersion = LanguageRegistry
-                .findLanguageVersionByTerseName(this.getLanguage() + ' ' + this.getVersion());
+                .findLanguageVersionByTerseName(forceLanguage != null ? forceLanguage : (this.getLanguage()) + ' ' + this.getVersion());
```

Comment:
that looks weird - if both -force-language and -language/-version is given, then -language/-version is ignored....